### ISSUE TITLE
Plot the graph to show the accuracy and loss from the training data

### ISSUE THREAD

**Issue by engichang1467** [engichang1467/Space-Invader-Keras-RL#2]
Use matplotlib and pandas to visualize our training data from ```trainingData.csv```
